release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -0,0 +1,72 @@
+
+### Shadow Selection
+
+Weeks: -1 to 1
+
+Once most of the Role Leads are selected, the EA should solicit candidates for Shadows.  Steps for this are:
+
+1. Get a copy of the Shadow Form (a Googledoc, currently) from the outgoing EA.
+2. Make any changes to it identified as required in the Retro or elsewhere.
+3. When the new RL gives the OK, open the form and announce it in the following places: #sig-release channel, sig-release mailing list, kubernetes-dev mailing list, and the Kubernetes Community Meeting.
+4. Generally the form will be open for 7-10 days.
+5. Chat with each of the incoming Role Leads about selecting their shadows, and make sure that they stay on schedule for it.  
+
+The most time-consuming part of this is helping the Role Leads select shadows.  In addition to the usual dilemmas of too many good candidates, the EA needs to give advice that makes sure that a diverse pool of shadows is selected, and that the Role Lead doesn't take on more shadows than they can effectively mentor.
+
+Occasionally, a role may not get enough qualified candidates, in which case it's the EA's job to work with the new Role Lead to find or solicit additional candidates.
+
+### Shadow Status
+
+Weeks: 1 to 12
+
+The apprenticeship program that is shadowing doesn't run itself.  The EA is responsible for a few things to make sure it is effective.
+
+First, have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected.  This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.
+
+Then, throughout the release cycle, do what you can to make sure that the shadows are participating in RT activities, particularly meetings.  For example, each Shadow should deliver the Role update during a release meeting or burndown meeting at least once. If a Role Lead is absent from a meeting and no Shadow takes over, that often indicates a problem.
+
+### Succession
+
+Weeks: 9 to 12
+
+Have meetings, or private chats, with each Role Lead and each Shadow Team to talk about succession.  For the Role Leads, find out who they would nominate to be the new lead and why, and record this information for the incoming Release Lead.  For the Shadows, ask them to think about what they want to do after the release.  This not only includes further RT roles, but other things they could do such as branch management or release engineering.
+
+The EA also needs to pick the EA for the next release no later than Week 11 (pending confirmation by the incoming RL), because they need to get started before final release.  
```

Comment:
```suggestion
The EA also needs to pick the EA for the next release no later than Week 11 (pending confirmation by the incoming Release Team Lead), because they need to get started before final release.  
```